spec.emu

```diff
@@ -138,9 +139,11 @@ copyright: false
   </dl>
   <emu-alg>
     1. Let _iterCount_ be the number of elements in _iters_.
+    1. Let _openIters_ be a copy of _iters_.
```

Comment:
I don't like this phrasing. Is there precedent? I would probably have written "Let \_openIters\_ be a new List whose elements are the elements of \_iters\_".

Reply:
There's precedent in a few places and also that specific phrase is suggested in [Algorithm Conventions](https://tc39.es/ecma262/multipage/notational-conventions.html#sec-algorithm-conventions):

> Algorithm steps may declare named aliases for any value using the form “Let x be someValue”. These aliases are reference-like in that both x and someValue refer to the same underlying data and modifications to either are visible to both. Algorithm steps that want to avoid this reference-like behaviour should explicitly make a copy of the right-hand side: “Let x be a copy of someValue” creates a shallow copy of someValue.